config.toml

```diff
@@ -80,7 +80,7 @@ languageCode = "en-us"
 contentDir = "content/en"
 weight = 1
 home = "Home"
-copyright = "Copyright &copy; 2022 Rotational Labs, LLC &middot; All Rights Reserved"
+copyright = "Copyright &copy; 2023 Rotational Labs, LLC &middot; All Rights Reserved"
```

Comment:
Could `new Date().getFullYear()` be added in the `site-footer` partial so that this doesn't have to be updated every year?

Reply:
Thank you so much for updating the year! If you could also update the name, that would be great -- we are now Rotational Labs, Inc 😉

Reply:
@daniellemaxwell I was thinking about that, but the script will be calling the dom at every page load that's why I just added the constant variable for now and thinking about that later.